.github/filters.yml

```diff
@@ -2,6 +2,30 @@ bandage_image:
   - software/bandage/image/**
   - tests/software/bandage/image/**
 
+bcftools_bgzip:
+  - software/bowtie/bgzip/**
+  - tests/software/bcftools/bgzip**
+
+bcftools_consensus:
+  - software/bowtie/consesus/**
```

Comment:
```suggestion
  - software/bcftools/consesus/**
```

Reply:
😆 Maybe why the tests aren't running?

Reply:
Solved here https://github.com/nf-core/modules/pull/134/commits/58d28ed896a79eb92d18d2371223f790ff361f17